management_dashboard/data/README_requesttypes.txt

```diff
@@ -0,0 +1,102 @@
+// From database sending user profile
+{
+    "type": "push profile",
+    "user_id": "1",
+    "challenge_id": ["1", "2"],
+    "user_name": "Mario",
+    "total_steps": "2200",
+    "remaining_sec": "2000",
+    "current_time": "2020-02-30T08:35:30.0108Z"
+}
```

Comment:
this is the old file; make sure to check the most recent one in github dev branch

Reply:
my bad, forgot about that one